packages/nc-gui/components/cell/MultiSelect.vue

```diff
@@ -150,7 +151,13 @@ watch(isOpen, (n, _o) => {
     @keydown="handleKeys"
     @click="isOpen = !isOpen"
   >
-    <a-select-option v-for="op of options" :key="op.id" :value="op.title" @click.stop>
+    <a-select-option
+      v-for="op of options"
+      :key="op.id"
+      :value="op.title"
+      :pw-data="`select-option-${column.title}-${rowIndex}`"
```

Comment:
`rowIndex` is not initialized

Reply:
@pranavxc 

We are passing rowIndex to both select cells

<img width="901" alt="image" src="https://user-images.githubusercontent.com/35809690/199695315-a90010d4-e8e8-48c3-90f1-4a32734c9113.png">

Reply:
Oh my bad, looks fine then :+1: